apis/kueue/v1beta1/workload_types.go

```diff
@@ -77,6 +77,18 @@ type WorkloadSpec struct {
 	Active *bool `json:"active,omitempty"`
 }
 
+type PodSetTopologyRequest struct {
+	// Required is the level label indicated by the `kueue.x-k8s.io/podset-required-topology` annotation
```

Comment:
```suggestion
	// required is the level label indicated by the `kueue.x-k8s.io/podset-required-topology` annotation
	// in jobs.
```

Reply:
Additionally, this annotation is added to Jobs, right?

Reply:
The annotation is added to PodSets, adjuted the comment to make it clear.